File: src/main/java/xyz/thewhitedog9487/ServerLifecycleListener.java
package xyz.thewhitedog9487;

import net.fabricmc.fabric.api.event.lifecycle.v1.ServerLifecycleEvents;

public class ServerLifecycleListener {
    public static void Register() {
        ServerLifecycleEvents.SERVER_STARTED.register(server -> {
            CommandRegister.OldPositions.clear();
            RandomTeleporter.LOGGER.info("已清空传送历史记录"); } ); } }

File: src/main/kotlin/xyz/thewhitedog9487/ServerLifecycleListener.kt
package xyz.thewhitedog9487

import net.fabricmc.fabric.api.event.lifecycle.v1.ServerLifecycleEvents

fun ServerLifecycleListenerRegister() {
    ServerLifecycleEvents.SERVER_STARTED.register { server ->
        OldPositions.clear()
        ModLogger.info("已清空传送历史记录") } }